packages/venia-ui/lib/resolvers/resolvers.js

```diff
@@ -1,6 +1,6 @@
-import { giftOptionsResolvers } from '../components/CartPage/PriceAdjustments/GiftOptions/giftOptions.gql';
-import { paymentInformationResolvers } from '../components/CheckoutPage/PaymentInformation/paymentInformation.gql';
 /**
+ * @deprecated Resolvers are deprecated in ApolloClient v3.
```

Comment:
I never know where to draw that line. 

1. This is going to be a major change anyway, right?
2. When do we actually remove this code?

Reply:
Funny enough I just spoke with Jimmy about this, but in regards to other deprecations (redux/context). While it sucks to leave dead code lying around, I think that we can and should solve for the removal at a later time. We've only just begun to use the deprecated indicator in jsdoc so I'd like to float them for a bit and come up with a removal "deadline". It might be "once @deprecated is added, feature/functionality will be removed by the next major" or it might be "by next march". 🤷

Reply:
1. Yes, it's absolutely going to be a major change.
2. We haven't actually formalized a policy for removal, only deprecation.

I think the pacing of our major releases is a bit too fast for us to change code "canonical" to "deprecated" to "removed" like that. Now that we're properly marking things with `@deprecated`, we can start the conversation around an actual removal timeframe.